docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[release-notes-header-8.8.0]]
+== 8.8
+
+[[release-notes-8.8.0]]
+=== 8.8.0
+
+To view a detailed summary of new features and enhancements we've documented this release, check out our {security-guide}/whats-new.html[8.8 release highlights].
+
+[discrete]
+[[known-issue-8.8.0]]
+==== Known issues
+
+* Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+
+[discrete]
+[[breaking-changes-8.8.0]]
+==== Breaking changes
+
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+* The privileges for attaching alerts to cases have changed. Now, you need at least `Read` privileges for Security and `All` privileges for Cases ({pull}147985[#147985]).
+* Adds conditional actions to the rules API. In {elastic-sec} 8.7 and earlier, action frequencies were set on a rule level by defining the `throttle` field. In 8.8 and later, action frequencies are set at the action level, and the `throttle` field is replaced by the `frequency` and `alert_filters` fields. The following APIs are affected:
+** <<rules-api-get,Get rule>>
+** <<rules-api-find,Find rules>>
+** <<rules-api-create,Create rule>>
+** <<rules-api-update,Update rule>>
+** <<bulk-actions-rules-api,Bulk rule actions>>
```

Comment:
Sorry I'm late with this comment, but I couldn't find any docs on what schema the `frequency` and `alert_filters` fields have. I think this section should be updated: https://www.elastic.co/guide/en/security/current/rules-api-create.html#actions-object-schema

Reply:
Hey, @banderror - no worries. I'm actually adding those docs in this PR: https://github.com/elastic/security-docs/pull/3320

Once I merge the PR, I can update these doc links to point to the `frequency` and `alert_filters` docs. Or, we can just keep them as is and point to the main page, for example:
- [Get rule](https://www.elastic.co/guide/en/security/8.8/rules-api-get.html)
- [Find rule](https://www.elastic.co/guide/en/security/8.8/rules-api-find.html)
- [Create rule](https://www.elastic.co/guide/en/security/8.8/rules-api-create.html)
- [Update rule](https://www.elastic.co/guide/en/security/8.8/rules-api-update.html)
- [Bulk rule actions](https://www.elastic.co/guide/en/security/8.8/bulk-actions-rules-api.html)

cc: @benironside @jmikell821

Reply:
Hey, @banderror. We've updated these links as per your request [here](https://www.elastic.co/guide/en/security/8.8/release-notes-header-8.8.0.html#breaking-changes-8.8.0). Thanks again for checking on this!

Reply:
Thanks, @nastasha-solomon!